File: Bottle.java

public class Bottle {
    public static void main(String[] args)
    {
        int numBeer = 99;
        String world = "bottles";
        while (numBeer > 0) {
            if (1 == numBeer) {
                world = "bottle";
            }
            System.out.println(numBeer + " " + world + " beer on the wall. Take one down.");
            numBeer -= 1;
            if (numBeer > 0) {
                System.out.println(numBeer + " " + world + " on the wall");
            } else {
                try {
                    Thread.sleep(20000);
                } catch (InterruptedException e) {
                    System.out.println(e);
                }
                System.out.println("No more bottles");
            }
        }
    }
}

File: bottle.php
<?php

function main() {
    $numBeer = 99;
    $world = "bottles";
    while ($numBeer > 0) {
        if ($numBeer == 1) {
           $world = "bottle";
        }
        echo $numBeer . " " . $world . "of beer on the wall. Take one down.\n";
        $numBeer -= 1;
        if ($numBeer > 0) {
            echo $numBeer . " " . $world . " of beer on the wall\n";
        } else {
           sleep(20);
           echo "No more bottles of beer on the wall\n";
        }
    }
}

main();
